### PR TITLE
[alchemy-vision] Add support for Text Detection. #213

### LINKED ISSUE #213: [alchemy-vision] Add support for Text Detection.
POST /image/ImageGetRankedImageSceneText Identifies text in the POSTed image
GET /url/UrlGetRankedImageSceneText Identifies text in the URL

The documentation is: http://www.ibm.com/smarterplanet/us/en/ibmwatson/developercloud/alchemyvision/api/v1/#

### PR DESCRIPTION
**Pull request by germanattanasio**
Fixes #213

### CODE REVIEW

**Comment by germanattanasio** [comment]
@nfriedly I will ask you to review my commits from now on.

File: test/test.integration-all-services.js [hunk 1]
```diff
@@ -605,6 +605,11 @@ describe('integration-all-services', function() {
       }, failIfError.bind(failIfError, done));
     });
 
+    it('getImageSceneText()', function(done) {
+      alchemy_vision.getImageSceneText({
+        url: 'http://itsopen.co.uk/site/wp-content/themes/its-open/assets/images/logo-its-open-social-media.png'
```
Comment (nfriedly) [inline test/test.integration-all-services.js]: Who owns that website? If it isn't us, then we should probably find somewhere else to host the image (I'm thinking we stick it in with the docs on the gh-pages branch)

**Comment by germanattanasio** [comment]
I created an app to host the resources we use in the integration tests and updated the test case to use that app.

Bluemix app: https://watson-test-resources.mybluemix.net/resources/obama.jpg
code: https://github.com/germanattanasio/watson-test-resources

@nfriedly feel free to update the code above and the Bluemix app (org: WatsonPlatformServices/ space: sdks)

**Comment by germanattanasio** [comment]
@nfriedly can you take a look ?

**Comment by nfriedly** [comment]
Right, sorry, this looks good. merging.

**Comment by nfriedly** [comment]
Should be released in v1.4.0 here shortly

**Pull request merged**